src/FakeItEasy/Sdk/Create.cs

```diff
@@ -0,0 +1,91 @@
+namespace FakeItEasy.Sdk
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using FakeItEasy.Creation;
+
+    /// <summary>
+    /// Provides methods for generating fake objects.
+    /// </summary>
+    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "A", Justification = "Is spelled correctly.")]
+    public static class Create
+    {
+        private static IFakeCreatorFacade FakeCreator
+        {
+            get { return ServiceLocator.Current.Resolve<IFakeCreatorFacade>(); }

```

Comment:
I just realized that `IFakeCreatorFacade` is not registered as a singleton... and most of its dependencies are not singletons either. This means that everytime we call `A.Something`, we create new instances of all these things. I wonder if this could be improved...